.travis.yml

```diff
@@ -226,6 +225,5 @@ after_success:
   - |
     set -e
     if [ "$DO_COVERAGE" = "coveralls" ]; then
-      cat build/luacov.stats.out
```

Comment:
Will change it to `test -f`, since this is where missing coverage info used to bail out.